migrations/v2.js

```diff
@@ -0,0 +1,134 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Resources - v2.0.1 to v2.0.2', async () => {
+  let course;
+  whereFromPlugin('Resources - from v2.0.1', { name: 'adapt-contrib-resources', version: '>=2.0.0 <2.0.2' });
+
+  whereContent('Resources - where Resources', async (content) => {
+    course = getCourse();
+    return course?._resources;
+  });
+
+  mutateContent('Resources - add _drawerOrder attribute', async (content) => {
```

Comment:
`_drawerOrder` added [in](https://github.com/adaptlearning/adapt-contrib-resources/releases/tag/v2.1.0) [2.1.0](https://github.com/adaptlearning/adapt-contrib-resources/commit/b2867cde080a326526e6fdd242a1641077b9cbcd#diff-bbbc797064913d6074c5515fadbef42b3c90e0b677002d065500e190e0d3bf88) rather than 2.0.2.